presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/ActualProperties.java

```diff
@@ -167,6 +187,13 @@ public ActualProperties translate(Function<Symbol, Optional<Symbol>> translator)
         return localProperties;
     }
 
+    public ActualProperties withReplicatedNulls(boolean replicatedNulls)
```

Comment:
Do you need the boolean arg, or is this always called with `true`.  Basically, should this be request to enable replicated nulls?